lib/connextclient/errors.ts

```diff
@@ -76,6 +77,10 @@ const errors = {
     message: 'seed is missing',
     code: errorCodes.MISSING_SEED,
   },
+  INSUFFICIENT_COLLATERAL: {
```

Comment:
Maybe it's also worth converting this error code to gRPC `status.FAILED_PRECONDITION` so that on the CLI level we get a nicer error message? We're already doing it for `connextErrorCodes.INSUFFICIENT_BALANCE`: https://github.com/ExchangeUnion/xud/blob/master/lib/grpc/getGrpcError.ts#L42

Reply:
Yes, this makes sense.